client/src/App.js

```diff
@@ -38,6 +43,7 @@ function App() {
 					<Route path="/register" component={Register} />
 					<Route path="/about-us-more" component={AboutUsMore} />
 					<Route path="/what-we-do" component={WhatWeDoPage} />
+					{/* <Route path="/what-we-do" component={WhatWeDoPage} /> */}
 
```

Comment:
can you remove these commented code if you no longer need it

Reply:
fixed